packages/components/Badge/src/BadgeTokens.ts

```diff
@@ -7,9 +7,9 @@ export const defaultBadgeTokens: TokenSettings<BadgeTokens, Theme> = () =>
   ({
     iconSize: 12,
     borderWidth: globalTokens.stroke.width10,
-    bottom: globalTokens.spacing.none,
-    right: globalTokens.spacing.none,
-    textMargin: globalTokens.spacing.xxs,
+    bottom: globalTokens.sizeNone,
+    right: globalTokens.sizeNone,
+    textMargin: globalTokens.size40,
```

Comment:
xxs is meant to map to size20.

Reply:
All values should default to web/desktop values except for android-specific files.

Reply:
It looks like all the components got updated with the old iOS/Android ramp rather than the old win32/desktop ramp. You can also see the conversion here: https://github.com/microsoft/fluentui-design-tokens/pull/36/files

For corner radius and stroke width we ran into a similar problem where there were two different ramps (ex. on mobile, corner.radius.extraLarge was defined as 120 but on win32 it was 80) so it wasn't obvious what to update a reference to corner.radius.extraLarge to. For corner radius/stroke width we ended up using the desktop mappings for everything, including mobile components. The reasoning was that up until the recent iOS/Android token work, iOS was actually referencing windows tokens, and so most components were built with the assumption that the windows values were being used - using the desktop conversion would cause less unwanted change than the using the mobile conversion.

I would prefer if we stayed consistent here and just updated everything with the desktop conversion, but also fine if we wanted to use the mobile conversion table for ios/android specific files. Eventually I think we should be going through each component to check that with all the tokens changes they still align with the designs.

Reply:
Hmm correct, missed that. 
I mapped everything to Android/iOS tokens, just found out that the mappings are different in Desktop platforms.   

I think let's be consistent and map the values to desktop just like corner radius/stroke width and eventually we should be going through each component to check that with all the tokens changes they still align with the designs for Mobile Platforms.